### DISCUSSION TITLE
UI лишняя кнопка

### DISCUSSION THREAD

**Discussion by kaavain** [cdek-it/widget#14] [Ideas]
Опять же, уже писал - продублирую здесь.
Правый-верхний угол, радио выбора типа доставки рядом с аккордеоном. Если в настройках виджета скрыты все, кроме одного типа доставки (в моем случае - доступны только ПВЗ), то единственная кнопка все равно присутствует, хотя нажать на нее нельзя и смысла она не имеет. Следует скрывать блок этого радио в случае если кнопка одна.

**Comment by vermorag** [comment]
Поскольку проблема не является критичной, обращений подобной тематики зарегистрировано мало, с 18.12.2023 по 10.01.2023 в компании СДЭК вводится запрет на выкладку новых версий модулей - данный запрос поставлен в план разработки на 2Q.2024. В случае увеличения количества обращений по данному вопросу - срок может быть пересмотрен.

**Comment by vermorag** [comment]
Исправлено в 3.8.0
